bin/cloudns_api.sh

```diff
@@ -634,6 +642,25 @@ function dump_zone {
   fi
 }
 
+function dump_all_zones {
+  do_tests
+  if [ "$#" -ne "1" ]; then
+    print_error "dumpallzones expects exactly one argument" && exit 1
+  fi
+
+  local OUTPUT_DIR="$1"
+  mkdir -p $OUTPUT_DIR
```

Comment:
Could you please do something like:
```
if ! mkdir -p ${OUTPUT_DIR} >/dev/null 2>&1; then
  print_error "Unable to create output directory ${OUTPUT_DIR}"
  exit 1
fi
```
otherwise you'll end up with an unhandled error, best case a bunch of output files in the current directory, worst case a bunch of errors about not being able to write to the current directory.

Thanks!

Reply:
Sure, I'll do that when I get some free time.